internal/resolution/manifest/maven.go

```diff
@@ -51,186 +51,105 @@ func NewMavenManifestIO() MavenManifestIO {
 }
 
 type MavenManifestSpecific struct {
-	Properties                 []PropertyWithOrigin
-	RequirementsWithProperties []resolve.RequirementVersion
-	RequirementsFromOtherPOMs  []resolve.RequirementVersion // Requirements that we cannot modify directly
+	Properties             []PropertyWithOrigin
```

Comment:
From what I understand, these properties don't include properties from parents, right?
Maybe add a comment to note this?